src/Microsoft.OData.Core/Json/JsonValueUtils.cs

```diff
@@ -52,7 +51,7 @@ internal static class JsonValueUtils
         /// <summary>
         /// Map of special characters to strings.
         /// </summary>
-        private static readonly string[] SpecialCharToEscapedStringMap = JsonValueUtils.CreateSpecialCharToEscapedStringMap();
+        private static readonly string[] SpecialCharToEscapedStringMap = CreateSpecialCharToEscapedStringMap();

```

Comment:
nit: This changed line is not consistent with line #39.

Reply:
Why changing it to a different style by removing JsonValueUtils prefix everywhere in this file? I believe the original prefixed style is the primary style for all other files in the odata.net project that is helpful for auto-complete / readability, even though it is not required for syntax.
I would recommend keeping the original style.